espnet2/bin/asr_decode.py

```diff
@@ -193,16 +193,19 @@ def recog(
                 hyp = nbest_hyps[n - 1]
                 assert isinstance(hyp, Hypothesis), type(hyp)
 
-                # remove sos/eos and get results
+                # remove sos/eos, blank and get results
                 token_int = hyp.yseq[1:-1].tolist()
+                token_int = list(filter(lambda x: x != 0, token_int))
```

Comment:
I see, thank you!! I think this shape is good.
Would you split "remove sos/eos, blank and get results" and put a comment such that It is assumed that `<blank>` symbol equal to 0 ?

Reply:
Done!